app/datatables/parent_object_datatable.rb

```diff
@@ -81,25 +81,6 @@ def data
   # rubocop:enable Rails/OutputSafety,Metrics/MethodLength
   # rubocop:enable Metrics/AbcSize
 
-  def extent_of_full_text(parent_object)
```

Comment:
this method already exists on the parent object model so it did not need transfered